opentracing/src/main/java/opentracing/Tracer.java

```diff
@@ -13,25 +13,41 @@
  */
 package opentracing;
 
+
 /**
- * Tracer is a simple, thin interface for Span creation.
+ * Tracer is a simple, thin interface for Span creation, and Span propagation into different transport formats.
  */
 public interface Tracer {
 
   /**
-   * Create, start, and return a new Span with the given `operationName`, all without specifying a
-   * parent Span that can be used to incorporate the newly-returned Span into an existing trace.
+   * Create, start, and return a new Span with the given `operationName`.
+   * An optional parent Span can be specified used to incorporate the newly-returned Span into an existing trace.
    *
    * <p>Example:
    * <pre>{@code
    * Tracer tracer = ...
-   * Span feed = tracer.startTrace("GetFeed");
-   * Span http = tracer.startTrace("HandleHTTPRequest")
+   * Span feed = tracer.startTrace("GetFeed", null);
+   * Span http = tracer.startTrace("HandleHTTPRequest", feed)
    *                   .setTag("user_agent", req.UserAgent)
    *                   .setTag("lucky_number", 42);
    * }</pre>
    */
-  Span startTrace(String operationName);
+  Span startSpan(String operationName, /* @Nullable */ Span parent);
```

Comment:
Do we have any examples of this?

Something like this is already happening [here](https://github.com/opentracing/opentracing-java/pull/11/files#diff-f202d93da32a2cfe723cbf28dbb95522R30) and  [here](https://github.com/opentracing/opentracing-java/pull/11/files#diff-a90b8918cb3610641d03a6d8bf014704R43).

But i'm unclear as to what its purpose or need is exposed higher up in `Tracer`?
When would the caller want to decide which Span implementation to use? (that seems like an anti-pattern).

If the Tracer can't decide which Span is required without first holding attributes it'll have, then I think the Builder pattern is more appropriate.